azure-pipelines.yml

```diff
@@ -157,11 +157,10 @@ jobs:
           Write-Host ("##vso[task.setvariable variable=NugetPackageVersion;]$tagVersion")
         displayName: Set NuGet Version to Tag Number
         condition: startsWith(variables['Build.SourceBranch'], 'refs/tags/')
-# Temporarily disable Set Xcode Version until Xcode 13.3.1 is available on Azure Pipelines (required for .NET MAUI RC)
-      # - task: CmdLine@2
-      #   displayName: 'Set Xcode Version'
-      #   inputs:
-      #     script: echo '##vso[task.setvariable variable=MD_APPLE_SDK_ROOT;]'/Applications/Xcode_$(XcodeVersion).app;sudo xcode-select --switch /Applications/Xcode_$(XcodeVersion).app/Contents/Developer
+      - task: CmdLine@2
+        displayName: 'Set Xcode Version'
```

Comment:
seems like we do not need it, and can use latest version by default

Reply:
Good to know! I think we should keep it until the next version of Xcode is released in case we need to leverage it again.